db/migrate/20220512142751_create_action_mailbox_tables.action_mailbox.rb

```diff
@@ -0,0 +1,14 @@
+# This migration comes from action_mailbox (originally 20180917164000)
+class CreateActionMailboxTables < ActiveRecord::Migration[6.0]
```

Comment:
why do we have this migration? it did not appear in other upgrades to Rails 6.

Reply:
I don't know what that migration is 😅